test/new-e2e/tests/agent-platform/common/test_client.go

```diff
@@ -58,6 +58,10 @@ func getPackageManager(vmClient e2eClient.VM) PackageManager {
 	if _, err := vmClient.ExecuteWithError("command -v apt"); err == nil {
 		return pkgmanager.NewAptPackageManager(vmClient)
 	}
+
+	if _, err := vmClient.ExecuteWithError("command -v yum"); err == nil {
+		return pkgmanager.NewYumPackageManager(vmClient)
+	}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Probably a very unlikely edge case we shouldn't worry about, but it's possible to have `yum` installed without it being used as the OS package manager.

Reply:
Ahaha that would be very silly 🤣 . I think it should not an issue since we chose the AMI we use in the test. And as long as we don't find a very strange AMI, that should never happen

Reply:
Yep, highly unlikely we hit it - but figured it was possible given how we were checking 😉